src/foam/nanos/auth/ChangePasswordView.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
```

Comment:
According to what I learned from Google lawyers, you shouldn't update copyright dates. The date should be from when the file was first created and then never updated. So, "Copyright 2019" really means "Copyright 2019 and beyond".